packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -211,7 +211,7 @@ export class InternalDropdownItem extends React.Component<InternalDropdownItemPr
         ...(styleChildren && {
           className: css(element.props.className, classes)
         }),
-        ...(this.props.role !== 'separator' && { ref: this.componentRef })
+        ...(this.props.role !== 'separator' && { role } && { ref: this.componentRef })
```

Comment:
Nitpick: could this also be written as `...(this.props.role !== 'separator' && { role, ref: this.componentRef })`?